tableau-specialist/1.0/modules/ROOT/pages/tableau-specialist-connector-examples.adoc

```diff
@@ -0,0 +1,957 @@
+http://www.mulesoft.org/schema/mule/tableau-specialist http://www.mulesoft.org/schema/mule/tableau-specialist/current/mule-tableau-specialist.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+http://www.mulesoft.org/schema/mule/file http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd">
+	<http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" doc:id="d784f0fb-88c2-42ac-a729-199268071485" >
+		<http:listener-connection host="0.0.0.0" port="8081" />
+	</http:listener-config>
+	<tableau-specialist:tableau-hyper-config name="Tableau_Specialist_Hyper_configuration" doc:name="Tableau Specialist Hyper configuration" doc:id="dd5a868e-8b70-4296-b4a4-b6c8c6479410" >
+		<tableau-specialist:connection telemetry="DO_NOT_SEND_USAGE_DATA_TO_TABLEAU"/>
+	</tableau-specialist:tableau-hyper-config>
+		<flow name="tableau-demo-create-hyper-file" doc:id="615f59de-2e3f-48fc-ae09-4f4ab893aaef" >
+		<http:listener doc:name="Listener" doc:id="1417bfaa-e778-465f-a95d-a7cea478d092" config-ref="HTTP_Listener_config" path="/create-hyper-file"/>
+		<tableau-specialist:create-hyper-file doc:name="Create hyper file" doc:id="cd6303c9-e42e-4d75-a1c1-9ce92054f4f0" config-ref="Tableau_Specialist_Hyper_configuration" hyperFileName="hyperFileDemo.hyper" schemaName="schemademo" tableName="tabledemo">
+			<tableau-specialist:columns>
+				<tableau-specialist:column-definition columnName="Customer ID" columnType="TEXT" columnNullability="NOT_NULLABLE" />
+				<tableau-specialist:column-definition columnName="Customer Name" columnType="TEXT" columnNullability="NOT_NULLABLE" />
+				<tableau-specialist:column-definition columnName="Loyalty Reward Points" columnType="BIG_INT" columnNullability="NOT_NULLABLE" />
+				<tableau-specialist:column-definition columnName="Segment" columnType="TEXT" columnNullability="NOT_NULLABLE" />
+			</tableau-specialist:columns>
+		</tableau-specialist:create-hyper-file>
+		<ee:transform doc:name="Transform Message" doc:id="d1415eb0-1fd0-4bef-aa60-0fee97b28e61">
+			<ee:message>
+				<ee:set-payload><![CDATA[%dw 2.0
+output application/json
+---
+payload]]></ee:set-payload>
+			</ee:message>
+		</ee:transform>
+	</flow>
+</mule>
+----
+
+=== Steps for Running This Example
+
+. Verify that your connector is configured.
+. Save the project.
+. From a web browser, test the application by entering `http://localhost:8081/create-hyper-file-from-csv`.
+
+
+[[insert-data-to-hyper-file-from-csv]]
+== Insert Data to Hyper File from CSV - Hyper Configuration
+
+This Mule flow creates a hyper table and populates the table with data from
+a CSV file. To use this flow, you must run the <<create-hyper-file, Create Hyper File>>
```

Comment:
I suggest for here and in similar paragraphs if time:

To use this flow, run the <<create-hyper-file, Create Hyper File>>
flow to create the hyper table and then use the `src/main/resources/customers.csv`
file to populate the table.